src/kernel/ring/modular-implem.h

```diff
@@ -353,7 +353,7 @@ namespace Givaro {
         __GIVARO_CONDITIONAL_TEMPLATE(E = Element, IS_FLOAT(E))
         inline std::ostream& write (std::ostream& s, const E& a) const
         {
-            return s << Caster<Residu_t>(a);
+            return s << Caster<typename make_signed_int<Storage_t>>(a);
```

Comment:
g++-8 -DHAVE_CONFIG_H -I. -I..  -I.. -I../src/kernel/system -I../src/kernel/memory -I../src/kernel/field -I../src/kernel/ring -I../src/kernel/integer -I../src/kernel/rational -I../src/kernel -I../src/library/poly1 -I../src/kernel/bstruct -I../src/library/tools -I../src/kernel/recint -I../src/kernel  -O2 -march=native -Wall -DNDEBUG -UDEBUG -DSTD_RECINT_SIZE=8 -g -O2 -MT test-regression.o -MD -MP -MF .deps/test-regression.Tpo -c -o test-regression.o test-regression.C
In file included from ../src/kernel/ring/givaro/givaro/modular-integral.h:24,
                 from ../src/kernel/ring/givaro/modular.h:21,
                 from test-regression.C:56:
../src/kernel/ring/givaro/givaro/givaro/modular-implem.h: In member function ‘std::ostream& Givaro::Modular_implem<_Storage_t, _Compute_t, _Residu_t>::write(std::ostream&, const E&) const’:
../src/kernel/ring/givaro/givaro/givaro/modular-implem.h:356:25: error: parse error in template argument list
             return s << Caster<typename make_signed_int<Storage_t>>(a);
                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~